content/docs/security/cors.md

```diff
@@ -67,9 +78,81 @@ Set up the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}} CRDs and the Pe
 
 ## Set up CORS policies
 
-Use the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}} CRDs to create CORS policies.
+Create a CORS policy for the Petstore app in an HTTPRoute or TrafficPolicy.
+
+{{< tabs items="CORS in HTTPRoute,CORS in TrafficPolicy" >}}
+{{% tab %}}
+Create an HTTPRoute resource for the Petstore app that applies a CORS filter. The following example allows requests from the `example.com/` and `*.example.com` origins.
```

Comment:
I'm not sure the `*.example.com` is a good example that will work with real hostnames (e.g. b.example.com).
We currently only do exact matches. I.e not wildcards prefix, suffix or regex although Envoy supports it.

Reply:
You are right, the wildcard is not working, thanks for the catch!

Reply:
@ymesika do you know if we intend to address this limitation in a future release?